specs/gloas/validator.md

```diff
@@ -104,11 +191,33 @@ alias `data = attestation.data`, the validator should set this field as follows:
   - Set `data.index = 1` to signal that the payload is present in the canonical
     chain (payload status is `FULL` in the fork-choice).
 
+#### Attestation data
+
+*Note*: The only change to `attestation_data` is to call
+`get_attester_head(store, head_root)` to set the `beacon_block_root` field of
+`attestation_data`.
+
+##### Modified LMD GHOST vote
+
+Set `attestation_data.beacon_block_root = get_attester_head(store, head_root)`.
+
 ### Sync Committee participations
 
 Sync committee duties are not changed for validators, however the submission
 deadline is changed with `SYNC_MESSAGE_DUE_BPS_GLOAS`.
 
+*Note*: The only change to `get_sync_committee_message` is to call
+`get_attester_head(store, head_root)` to set the `beacon_block_root` parameter
+of `get_sync_committee_message`.
+
+#### `get_sync_committee_message`
+
+##### Modified `beacon_block_root`
+
+The `beacon_block_root` parameter MUST be set to return value of
+[`get_attester_head(store: Store, head_root: Root)`](./fork-choice.md#new-get_attester_head)
+function.
```

Comment:
On line 218, I don't think we should link to fork-choice specs. We don't normally do this & it's strange that we picked this spot to do it since it's used earlier. We don't need to include the parameter types either.